docs/reference/commandline/swarm_leave.md

```diff
@@ -31,7 +31,7 @@ dkp8vy1dq1kxleu9g4u78tlag    worker1   Ready   Active        Reachable
 dvfxp4zseq4s0rih1selh0d20 *  manager1  Ready   Active        Leader
 ```
 
-On a worker node:
+On worker2 node:
```

Comment:
SGTM @thaJeztah , recommend comma over parens.

On a worker node, `worker2` in the following example: